flake.nix

```diff
@@ -23,6 +23,7 @@
           pkgs.nodejs_22
           pkgs.nodePackages.pnpm
           pkgs.bun
+          pkgs.direnv
```

Comment:
Quesiton: I haven't used direnv, but my understanding is that you need it installed outside of the flake so that it knows to activate the flake in the first place? If so, does it make sense to also include it as a dep inside the flake?

Reply:
I'm not fully sure @shazow. I've always had it installed globally.

Reply:
IMO it shouldn't be in here, it's kinda like including `nix` in here. :P

Reply:
Yea agreed once you are in nix this becomes useless since you are already in mix

Reply:
> IMO it shouldn't be in here, it's kinda like including `nix` in here. :P

this is a typical usage of `direnv` in nix. What do you mean like including `nix`?

One use case of `direnv` is you don't have to run `nix develop` if you use it. Also, your globally installed shell utilities remain available.

Reply:
That's what I mean, `direnv` already needs to be installed for this flake to be loaded, so it's moot to have the flake install it again? Similar to have `nix` already needs to be installed for this flake to be loaded.

Unless I don't understand how direnv works correctly. :)

Anyway it's not hurting anything, I just don't think it's helping anything either.

Reply:
yes good point. What I need to do is add instructions to getting started

```
# for direnv usage
nix shell nixpkgs#direnv
# then
direnv allow
```